src/smartpeak/include/SmartPeak/pipelines/LCMS_MRM_Unknown_example.h

```diff
@@ -59,6 +59,7 @@ void example_LCMS_MRM_Unknowns(
   for (const InjectionHandler& injection : sequenceHandler.getSequence()) {
     const std::string& key = injection.getMetaData().getInjectionName();
     dynamic_filenames[key] = Filenames::getDefaultDynamicFilenames(
+      dir_I,
```

Comment:
I am assuming that this is for the users SmartPeak directory where all of their input files are located?

Reply:
Yes. I need to be able to read that single file ("static") from the sampleGroupProcessor, which was never needed before in any other processor. So I had to introduce that directory. i was not sure to it the right way, but I haven't found any other example.

While doing this, I was thinking the Filenames class should be little bit reworked. I feel it's uncomfortable to have to declare the default file names in that Filename class when you add one processor. the processor should be a consistent, self declarative unit ("declarative" is important there: the Filename class can still have some responsibilities). in a way the processor should declare to the Filename class the file that it will need. and the Filename would construct the full path and check existence etc.. something like this. a JIRA task has been created for that we can discuss further.